app/pages/search/controls/PeopleCapacityControl.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import FontAwesome from 'react-fontawesome';
+import NumericInput from 'react-numeric-input';
+
+import { injectT } from 'i18n';
+import MiniModal from 'shared/mini-modal';
+
+PeopleCapacityControl.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  t: PropTypes.func.isRequired,
+  value: PropTypes.number,
+};
+
+function PeopleCapacityControl({ onChange, t, value }) {
+  return (
+    <div className="app-PeopleCapacityControl">
+      <MiniModal
+        buttonContent={
+          <span><FontAwesome name="users" /> {value || ''}</span>
```

Comment:
- [ ] I don't see this icon in the browser

I've tried with other font-awesome icons and I don't see them either.